interop-testing/src/test/java/io/grpc/testing/integration/Http2NettyLocalChannelTest.java

```diff
@@ -33,19 +33,20 @@
  * Run transport tests over the Netty in-process channel.
  */
 @RunWith(JUnit4.class)
-public class Http2NettyLocalChannelTest extends AbstractInteropTest {
+public class Http2NettyLocalChannelTest extends AbstractNettyInteropTest {
 
   private DefaultEventLoopGroup eventLoopGroup = new DefaultEventLoopGroup();
 
   @Override
-  protected AbstractServerImplBuilder<?> getServerBuilder() {
-    return NettyServerBuilder
-        .forAddress(new LocalAddress("in-process-1"))
-        .flowControlWindow(AbstractInteropTest.TEST_FLOW_CONTROL_WINDOW)
-        .maxInboundMessageSize(AbstractInteropTest.MAX_MESSAGE_SIZE)
-        .channelType(LocalServerChannel.class)
-        .workerEventLoopGroup(eventLoopGroup)
-        .bossEventLoopGroup(eventLoopGroup);
+  protected ServerBuilder<?> getServerBuilder() {
+    return withCustomCensusModule(
```

Comment:
This sort of method seems volatile and each time it changes it would change indentation for those builder methods. Let's just define a local variable for the builder and call withCustomCensusModule() on a separate line.